gulpfile.js

```diff
@@ -1228,7 +1248,13 @@ function buildSandcastle() {
         })
         .pipe(gulp.dest('Build/Apps/Sandcastle'));
 
-    return streamToPromise(mergeStream(appStream, imageStream));
+    var standaloneStream = gulp.src([
+        'Apps/Sandcastle/standalone.html'
+        ])
+        .pipe(gulpReplace('../../ThirdParty/requirejs-2.1.20/require.js', '/Build/CesiumUnminified/Cesium.js'))
```

Comment:
Here's one suspect path.  It looks server-root-relative.

Reply:
Okay yeah, I can recreated it locally now.